eel/eel-labeled-image.c

```diff
@@ -1102,8 +1102,13 @@ labeled_image_update_alignments (EelLabeledImage *labeled_image)
 
         if (labeled_image->details->fill)
         {
+#if GTK_CHECK_VERSION (3, 16, 0)
+            gtk_label_get_xalign (GTK_LABEL (labeled_image->details->label));
```

Comment:
Both ```gtk_label_get_xalign``` and ```gtk_label_get_yalign``` actually return values. Given that this code block should do the same thing as ```gtk_misc_get_alignment``` does with GTK+ < 3.16, I think it should look like this:

```C
x_alignment = gtk_label_get_xalign (GTK_LABEL (labeled_image->details->label));
y_alignment = gtk_label_get_yalign (GTK_LABEL (labeled_image->details->label));
```